demo-transfer-dashboard/src/Components/Form.jsx

```diff
@@ -1,75 +1,104 @@
-import React, { useState } from 'react';
+import { useState, useEffect } from 'react';
 import {
   GcdsInput,
-  GcdsText,
-  GcdsButton,
   GcdsSelect,
+  GcdsButton,
   GcdsErrorMessage,
+  GcdsText,
 } from '@cdssnc/gcds-components-react';
 
 const Form = () => {
   const [patientNumber, setPatientNumber] = useState('');
   const [originPT, setOriginPT] = useState('BC');
   const [receivingPT, setReceivingPT] = useState('ON');
-  const [errorMessage, setErrorMessage] = useState('');
-  const [loadingMessage, setLoadingMessage] = useState(false);
   const [transfers, setTransfers] = useState([]);
+  const [loadingMessage, setLoadingMessage] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
 
-  const handlePatientNumberChange = (e) => {
-    setPatientNumber(e.target.value);
-  };
+  useEffect(() => {
+    fetchTransfers();
+  }, []);
 
-  const handleOriginPTChange = (e) => {
-    setOriginPT(e.target.value);
-  };
+  const fetchTransfers = async () => {
+    try {
+      setLoadingMessage('Loading transfer requests...');
+      const response = await fetch(
+        `${process.env.BC_OUTBOUND_URL}/transfer-request`,
+      );
+      if (!response.ok) throw new Error('Failed to load transfers');
 
-  const handleReceivingPTChange = (e) => {
-    setReceivingPT(e.target.value);
+      const data = await response.json();
+      console.log(data);
+      setTransfers(data);
```

Comment:
Should do a separate request for ON transfers as well.

The responses don't include the outbound code since the source of truth for that should be the service URL used, so if you want to include the outbound province in the display you could add it here after the response(s).